### PR TITLE
Fix compile time and runtime issues with testapp

### PR DESCRIPTION
**Pull request by lbegani**

### CODE REVIEW

**Comment by lbegani** [comment]
Fixes the issue https://github.com/Dronecode/camera-manager/issues/173 and https://github.com/Dronecode/camera-manager/issues/172
@hamishwillee

File: test/test_mavlink_protocol.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * This file is part of the Camera Streaming Daemon project
+ * This file is part of the Dronecode Camera Manager
```
Comment (shakthi-prashanth-m) [inline test/test_mavlink_protocol.cpp]: I suggest to add a brief description of what the app does. Sample [here](https://github.com/dronecore/DroneCore/blob/3cd51ed0b356d1eb5cac1d785104ef61ab3699d3/example/fly_qgc_mission/fly_qgc_mission.cpp#L1-L25).

**Comment by julianoes** [comment]
@lbegani should this still go in?

File: test/test_mavlink_protocol.cpp [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 /**
 
-@brief  This is  a test application to test mavlink messages in the Camera Streaming Daemon.
+@brief  This is  a test application to test mavlink imessages in the Dronecode Camera Manager.
```
Comment (hamishwillee) [inline test/test_mavlink_protocol.cpp]: imessages ?

**Comment by lbegani** [comment]
> @lbegani should this still go in?

For now, we can close this by merging it. Improvements can go in later.

**Comment by julianoes** [comment]
To me it looks like @shakthi-prashanth-m 's comments should be addressed though. I might add them before merging.

**Pull request merged**